LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -106,17 +106,15 @@ public async Task<Twin> GetTwinAsync(CancellationToken cancellationToken = defau
             }
         }
 
-        public async Task<bool> UpdateReportedPropertiesAsync(TwinCollection reportedProperties)
+        public async Task<bool> UpdateReportedPropertiesAsync(TwinCollection reportedProperties, CancellationToken cancellationToken)
         {
             try
             {
-                this.deviceClient.OperationTimeoutInMilliseconds = 120000;
-
                 SetRetry(true);
 
                 this.logger.LogDebug("updating twin");
 
-                await this.deviceClient.UpdateReportedPropertiesAsync(reportedProperties);
+                await this.deviceClient.UpdateReportedPropertiesAsync(reportedProperties, cancellationToken);
```

Comment:
we kind of rely on the fact that the cancellationToken is not None now, should we enforce it in here?

Reply:
As discussed, I am now enforcing a default timeout if cancellation token is none